### PR TITLE
Add sars anvil tutorial

### PR DESCRIPTION
**Pull request by nakucher**
Excited to contribute the GDSCN SARS with Galaxy in AnVIL (https://jhudatascience.org/GDSCN_Book_SARS_Galaxy_on_AnVIL/index.html) tutorial to the GTN! This is my first tutorial so please feel free to help me fix anything that needs fixing.

### CODE REVIEW

File: topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,424 @@
...
+> >
+> > 1. Alignment tools figure out the optimal positioning of the reads next to the reference genome to minimize mismatches and gaps. 
+> > 2. Paired data is made up of fragments that are read twice (forward and reverse). Paired data improves data accuracy.
+> >
+> {: .solution}
+{: .question}
+
+# Viewing aligned data
+
+We have aligned our data but it is currently a table of where the reads align. This is hard to read, so we will use [JBrowse](https://jbrowse.org/jb2/) to view the data.
```
Comment (hexylena) [inline topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md:325]:
```suggestion
We have aligned our data but it is currently a table of where the reads align. This is hard to read, so we will use [JBrowse](https://jbrowse.org/jbrowse1.html) to view the data.
```

jb2 is still work in progress, not in galaxy yet.

Comment (hexylena) [reply topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md:325]: very cool to see y'all using it though! Really great to see

File: topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,424 @@
...
+> 1. Return to AnVIL, and find the Galaxy logo that shows your cloud environment is running. Click on the Galaxy logo.
+>   ![Screenshot of the Workspace menu. The currently running Galaxy cloud environment logo on the right of the page is highlighted.](/topics/sequence-analysis/images/sars-with-galaxy-in-anvil/19-galaxy-env.png)
+> 2. Click "Settings".
+>   ![Screenshot of the Environment details menu. The Settings button is highlighted.](/topics/sequence-analysis/images/sars-with-galaxy-in-anvil/20-galaxy-edit.png)
+> 3. Next, scroll down and click on “DELETE ENVIRONMENT”:
+>   ![Screenshot of the cloud environment pop out menu. The “DELETE ENVIRONMENT” button is highlighted.](/topics/sequence-analysis/images/sars-with-galaxy-in-anvil/21-galaxy-delete.png)
+> 4. Finally, select “Delete everything, including persistent disk”. Make sure you are done with the activity and then click “DELETE”.
+    ![Screenshot of the cloud environment pop out menu. The “Delete everything, including persistent disk” radio button has been checked and is highlighted. The “DELETE” button is highlighted.](/topics/sequence-analysis/images/sars-with-galaxy-in-anvil/22-galaxy-delete-pd.png)
```
Comment (hexylena) [inline topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md:419]:
```suggestion
>    ![Screenshot of the cloud environment pop out menu. The “Delete everything, including persistent disk” radio button has been checked and is highlighted. The “DELETE” button is highlighted.](/topics/sequence-analysis/images/sars-with-galaxy-in-anvil/22-galaxy-delete-pd.png)
```

File: topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,424 @@
...
+    editing:
+    - katherinecox
+    - cutsort
+    - nakucher
+    infrastructure:
+    - cansavvy
+    - avahoffman
+    - katherinecox
+    - ehumph
+    - cutsort
```
Comment (hexylena) [inline topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md:33]: if you have funding source you'd like to call out, you're welcome to annotate that here as well!

**Review by hexylena — COMMENTED** [review]
Hey @nakucher, welcome to the GTN! Great to see this. I've made some comments, and there are some more comments from the review bot that you'll find in the 'files' tab.

File: topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md [hunk 4]
```diff
@@ -0,0 +1,424 @@
...
+{: .warning}
+
+> <hands-on-title>Clone the Workspace </hands-on-title>
+>
+> 1. In your new Workspace, click on the “ANALYSES” tab. Next, click on “START”. You should see a popup window on the right side of the screen. 
+> ![Screenshot of the Workspace Notebooks tab. The notebook tab name and the plus button that starts a cloud environment for Galaxy have been highlighted.](/topics/sequence-analysis/images/sars-with-galaxy-in-anvil/05-start-galaxy.png)
+> 2. Click on the Galaxy logo to proceed.
+>    - Click on “NEXT” and “CREATE” to keep all settings as-is.
+>    - ![The NEXT button among cloud environments has been highlighted.](/topics/sequence-analysis/images/sars-with-galaxy-in-anvil/06-galaxy-next.png)
+>    - ![The CREATE button among cloud environments has been highlighted.](/topics/sequence-analysis/images/sars-with-galaxy-in-anvil/07-galaxy-create.png)
```
[the remark this answers is not in the document]
Comment (nakucher) [reply topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md:173]: thank you! @avahoffman is amazing at these!

File: topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md [hunk 5]
```diff
@@ -0,0 +1,424 @@
...
+lang: "en"
+---
+
+There is a growing need for undergraduate students to learn cutting-edge concepts in genomics data science, including performing analysis on the cloud instead of a personal computer. 
+
+This lesson aims to introduce a mutant detection bioinformatics pipeline based on a publicly available genetic sample of SARS-CoV-2. Students will be introduced to the sequencing revolution, variants, genetic alignments, and essentials of cloud computing prior to the lab activity. During the lesson, students will work hands-on with the point-and-click Galaxy interface on the AnVIL cloud computing resource to check data, perform an alignment, and visualize their results.
+
+> <agenda-title></agenda-title>
+> 
+> In this tutorial we will deal with:
```
[the remark this answers is not in the document]
Comment (nakucher) [reply topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md:46]: updated, thanks!

File: topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md (hunk 3 above)
Comment (nakucher) [reply topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md:33]: added, thanks!

File: topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md [hunk 6]
```diff
@@ -0,0 +1,424 @@
...
+> > 1. 43,522 reads. Zero flagged as poor quality. Sequence length is 3-301 base pairs.
+> > 2. 28-40 is pretty good. 
+> > 3. Yes, because the per base sequence quality is good (“in the green”).
+> >
+> {: .solution}
+{: .question}
+
+> <details-title>Learn more about quality scores</details-title>
+>
+> You may be wondering how the fourth line of the .fastq files relates to the quality score above. To save space, the sequencer records an [ASCII character](http://drive5.com/usearch/manual/quality_score.html) to represent scores 0-42. For example 10 corresponds to “+” and 40 corresponds to “I”. FastQC knows how to translate this. This is often called “Phred” scoring.
```
[the remark this answers is not in the document]
Comment (nakucher) [reply topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md:273]: linked to https://training.galaxyproject.org/training-material/topics/sequence-analysis/tutorials/quality-control/faqs/.

**Comment by nakucher** [comment]
@hexylena I am getting this error in the Accessibility Linting regarding the time_estimation `tutorial.md` metadata (this also happened as I tested building the site with gitpod). I've tried copy/pasting directly from other tutorials to match the formatting with no luck. Any ideas?

`  Liquid Exception: undefined method `[]' for nil:NilClass in /home/runner/work/training-material/training-material/_layouts/tutorial_hands_on.html
bundler: failed to load command: jekyll (/home/runner/work/training-material/training-material/vendor/bundle/ruby/3.0.0/bin/jekyll)
/home/runner/work/training-material/training-material/_plugins/jekyll-duration.rb:34:in `duration_to_human': undefined method `[]' for nil:NilClass (NoMethodError)`

File: topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md [hunk 7]
```diff
@@ -0,0 +1,424 @@
...
+>   - Click “CLONE WORKSPACE”.
+> 
+{: .hands_on}
+
+## Starting Galaxy
+
+Galaxy is a great tool for performing bioinformatics analysis without having to update software or worry too much about coding. In order to use Galaxy, we need to create a cloud environment. This is like quickly renting a few computers from Google as the engine to power our Galaxy analysis. 
+
+> <warning-title>Internet Browser</warning-title>
+> Currently, you will need to use Chrome or Safari as your browser for Galaxy cloud environments to work.
```
[the remark this answers is not in the document]
Comment (nakucher) [reply topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md]: Google Chrome is the browser where AnVIL is most supported. Safari and Firefox are still usable (afaik), but there might be some functionality that does not work as expected. I updated this text to reflect that instead!

**Comment by nakucher** [comment]
@hexylena all checks passed! would it be possible to merge?

File: topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md [hunk 8]
```diff
@@ -0,0 +1,412 @@
...
+
+On the welcome page, there are links to tutorials. You may try these out on your own. If you want to try a new analysis this is a good place to start.
+
+# Exercise One: Importing Data into Galaxy
+
+Luckily, we linked to the original data when we cloned our Workspace! We have three files we will need for our activity. These are (1) the reference genome for SARS-CoV-2, and both forward (2) and reverse (3) reads for our sample. There are two sets of reads for our sample because the scientists who collected it used paired-end sequencing. The reference genome ends in “.fasta” because it has already been cleaned up by scientists. The sample we are looking at ends in ".fastq" because it is raw data from the sequencer.
+
+> <hands-on-title>Import Data from the Workspace</hands-on-title>
+>
+> 1. Click on Upload Data in the Tools pane.
```
Comment (hexylena) [inline topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md:193]: Is tutorial mode not enabled in anvil environments? Ah well

File: topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md [hunk 9]
```diff
@@ -0,0 +1,412 @@
...
+> 2. Click on “Choose remote files” at the bottom of the popup. Double-click the top selection, which is the Workspace folder, then “Tables/” then “reference/”. Click the reference .fasta file so that it is highlighted in green and click “Ok”.
+> ![Screenshot of the Galaxy Data upload page with the current AnVIL workspace highlighted.](../../images/sars-with-galaxy-in-anvil/11-galaxy-data-workspace.png)
```
Comment (hexylena) [inline topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md:195]:
```suggestion
>    ![Screenshot of the Galaxy Data upload page with the current AnVIL workspace highlighted.](../../images/sars-with-galaxy-in-anvil/11-galaxy-data-workspace.png)
```

File: topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (hexylena) [reply topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md:264]: Ah not quite what I meant, I'll make some commits on top of this PR.

File: topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md [hunk 10]
```diff
@@ -0,0 +1,412 @@
...
+> <details-title>Learn more about quality scores</details-title>
+>
+> You may be wondering how the fourth line of the .fastq files relates to the quality score above. To save space, the sequencer records an [ASCII character](http://drive5.com/usearch/manual/quality_score.html) to represent scores 0-42. For example 10 corresponds to “+” and 40 corresponds to “I”. FastQC knows how to translate this. This is often called “Phred” scoring.
+> What does 0-42 represent? These numbers, when plugged into a formula, tell us the probability of an error for that base. This is the formula, where Q is our quality score (0-42) and P is the probability of an error:
+>
+>Q = -10 log10(P)
+>
+>Using this formula, we can calculate that a quality score of 40 means only 0.00010 probability of an error!
+>
+> Learn more from the [Quality Control Tutorial FAQs](https://training.galaxyproject.org/training-material/topics/sequence-analysis/tutorials/quality-control/faqs/).
```
Comment (hexylena) [inline topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md:271]: This should have triggered an alert but somehow it avoided it. Need to investigate that at some later date.

File: topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md [hunk 11]
```diff
@@ -0,0 +1,412 @@
...
+{: .details}
+
+# Exercise Three: Alignment
+
+Given that our data has passed some quality checks, we will try to align the data to the reference genome. In this case it is simple, a viral genome. A human sequencing project will generate much larger data sets. There are many aligners, but we will start off looking at a simple aligner BWA-MEM. This example uses paired data.
+
+We will use our two SARS data files, which are ready for alignment.
+
+- VA_sample_forward_reads.fastq
+- VA_sample_reverse_reads.fastq
```
Comment (hexylena) [inline topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md:282]:
```suggestion
- `VA_sample_forward_reads.fastq`
- `VA_sample_reverse_reads.fastq`
```

File: topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md [hunk 12]
```diff
@@ -0,0 +1,412 @@
...
+> {: .solution}
+{: .question}
+
+# Viewing aligned data
+
+We have aligned our data but it is currently a table of where the reads align. This is hard to read, so we will use [JBrowse](https://jbrowse.org/jbrowse1.html) to view the data.
+
+> <hands-on-title>Visualize Reference Data</hands-on-title>
+>
+> 1. Scroll down in the Tools menu to STATISTICS AND VISUALIZATION. Under "Graph/Display Data", select {% tool [JBrowse](https://toolshed.g2.bx.psu.edu/view/iuc/jbrowse/a6e57ff585c0) %}.
```
Comment (hexylena) [inline topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md:317]: This also should have failed linting, I really don't understand :| 

This isn't the right format for a tool link, it should look like 
```suggestion
> 1. Scroll down in the Tools menu to STATISTICS AND VISUALIZATION. Under "Graph/Display Data", select {% tool [JBrowse](toolshed.g2.bx.psu.edu/repos/iuc/jbrowse/jbrowse/1.16.9+galaxy0) %}.
```

File: topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md [hunk 13]
```diff
@@ -0,0 +1,412 @@
...
+
+You will need to click on the magnifying glasses to zoom in, but you should see the A,C,G, and Ts and their corresponding colors that make up the SARS-CoV-2 genome!
+
+![Screenshot of preliminary JBrowse results. The eye icon is highlighted, as it should be used to open the JBrowse viewer. The magnifying glasses in JBrowse are also highlighted as they enable zooming in to see the individual bases.](../../images/sars-with-galaxy-in-anvil/15-galaxy-jbrowse-ref.png)
+
+This is interesting, but it doesn’t let us compare the genome to the sample we have. We suspect there may be some differences that indicate our sample is the delta variant. 
+
+> <hands-on-title>Visualize Aligned Data</hands-on-title>
+>
+> 1. Scroll down in the Tools menu to STATISTICS AND VISUALIZATION. Under "Graph/Display Data", select {% tool [JBrowse](https://toolshed.g2.bx.psu.edu/view/iuc/jbrowse/a6e57ff585c0) %}.
```
Comment (hexylena) [inline topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md:333]:
```suggestion
> 1. Scroll down in the Tools menu to STATISTICS AND VISUALIZATION. Under "Graph/Display Data", select {% tool [JBrowse](toolshed.g2.bx.psu.edu/repos/iuc/jbrowse/jbrowse/1.16.9+galaxy0) %}.
```

**Review by hexylena — COMMENTED** [review]
Some comments :)

File: topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md [hunk 14]
```diff
@@ -0,0 +1,412 @@
...
+## Alignments
+
+This lecture module introduces the structure of genomic data and how alignments work. It touches on the “shredded book” analogy, demonstrates how short chunks of data can be compared to find variation, and reviews data files needed for alignments, including reference genomes and read data. It also reviews some diverse applications of variant detection made possible via alignment tools.
+
+Learning Objectives
+- Learn about data as “reads” & shredded books
+- Become familiar with reference genomes and alignments
+- Explore the file structure of genomic data and quality scoring
+
+<a href="http://www.youtube.com/watch?feature=player_embedded&v=MEZP_AzlLyg" target="https://youtu.be/MEZP_AzlLyg"><img src="http://img.youtube.com/vi/MEZP_AzlLyg/0.jpg" alt="Video thumbnail image for the Alignments background video." width="240" height="180" border="10" /></a>
```
[the remark this answers is not in the document]
Comment (nakucher) [reply topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md:87]: Totally agree! I'd love to do that in the near future.

File: CONTRIBUTORS.yaml [hunk 15]
```diff
@@ -1782,6 +1815,24 @@ elixir-converge:
...
       ELIXIR CONVERGE is connecting and align ELIXIR Nodes to deliver sustainable FAIR life-science data management services. This project has received funding from the European Union's Horizon 2020 research and innovation programme under grant agreement № 871075
 
+nhgri-gdscn:
+    name: National Human Genome Research Institute Genomic Data Science Community Network
+    github: false
+    joined: 2023-06
+    avatar: https://www.genome.gov/themes/custom/nhgri/assets/global/NHGRI-logo.svg
+    url: https://www.genome.gov/
+    funder: true
+    funding_id: 75N92022P00232
```
[the remark this answers is not in the document]
Comment (nakucher) [reply CONTRIBUTORS.yaml:1825]: Hmm yes it looks like RePORTER has been down for a little while :(

**Comment by hexylena** [comment]
Ok good it's failing now :smiling_imp: (sorry, but it should've been detecting the 'bad' links!)

**Review by hexylena — APPROVED** [review]

**Pull request merged**